style/javascript/README.md

```diff
@@ -0,0 +1,22 @@
+JavaScript
+==========
+
+[Sample](sample.js)
+
+* Prefer ES6 classes over library provided classes.
+* Prefer `===` and `!==` to `==` and `!=`.
```

Comment:
The use cases for `==` are very specific (IMO). Should we just specify them:

* Use strict equality checks (`===` and `!==`) except when comparing against (`null` or `undefined`).